tests/e2e/tests/content-manager/preview.spec.ts

```diff
@@ -69,4 +69,35 @@ describeOnCondition(edition === 'EE')('Preview', () => {
     // Expect the preview tab to be disabled (since the document is in draft status)
     await expect(page.getByText(/^Published$/)).toBeDisabled();
   });
+
+  test.only('Iframe should be present and load the correct URL', async ({ page }) => {
+    // Open an edit view for a content type that has preview
+    await clickAndWait(page, page.getByRole('link', { name: 'Content Manager' }));
+    await clickAndWait(page, page.getByRole('link', { name: 'Article' }));
+    await clickAndWait(page, page.getByRole('gridcell', { name: /west ham post match/i }));
+
+    // Publish the document
+    await clickAndWait(page, page.getByRole('button', { name: /publish/i }));
```

Comment:
I'm not sure clickAndWait does anything special if the click isn't a page navigation